vscode-extension/src/extension.ts

```diff
@@ -173,3 +176,51 @@ async function promptForAssemblyFilePathViaDialog(): Promise<string> {
         return undefined;
     }
 }
+
+function parseAssemblyPath(assemblies: string[]): AssemblyPathInfo[] {
+    const workspaceFolders = vscode.workspace.workspaceFolders;
+    return assemblies.map(assemblyPath => {
+        const p = path.parse(assemblyPath);
+        const assemblyWorkspace = workspaceFolders
+          .find(w => p.dir.includes(w.uri.fsPath));
+        return {
+          fileName: p.base,
+          fileExtension: p.ext,
+          fullPath: assemblyPath,
+          relativePath: p.dir.replace(assemblyWorkspace.uri.fsPath, ''),
+          workspaceFolder: assemblyWorkspace.name
+        };
+    });
+}
+
+function createAssemblyQuickPickItem(assemblyPathInfo: AssemblyPathInfo): vscode.QuickPickItem{
+    const selectIcon = (extension: string) => {;
```

Comment:
extraneous `;` at end it looks.
```suggestion
    const selectIcon = (extension: string) => {
```